test/analyzers/lint_analyzer/rules/rules_list/prefer_single_widget_per_file/examples/correct_example.dart

```diff
@@ -0,0 +1,8 @@
+import 'flutter_defines.dart';
+
+class someWidget extends StatelessWidget {
```

Comment:
Please add an example for stateful widget too.

Reply:
done